src/utils/hooks/useScroll.ts

```diff
@@ -0,0 +1,26 @@
+import { useEffect } from 'react';
```

Comment:
Please remove all additional code - only scroll top feature for the browser window is needed here. So all target, option, scrollToTarget should be removed. 
Rename this hook to useScrollReset and change this in all places